### PR TITLE
Badge button

### PR DESCRIPTION
**Pull request by shashikantmaurya0007**
https://www.loom.com/share/17fa283b3c9c4e9cbcb167e3174be3c9

feature:added button and badge component

### CODE REVIEW

**Review by kotesh-arya — APPROVED** [review]
Looks good to merge 👍🏻

File: pageLayout.css [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+@import url("https://cdn.jsdelivr.net/npm/bootstrap-icons@1.3.0/font/bootstrap-icons.css");
+:root {
+    --vodka-UI-background-color: white;
+    --vodka-UI-heading-color: #333333;
+    --vodka-UI-text-color: #525252;
+    font-family: Roboto;
+    --highlight-word: black;
+}
+
+::-webkit-scrollbar {
```
Comment (SukanyaSen530) [inline pageLayout.css:11]: Suggestion - if you are changing the scrollbar of a section, add the section's class otherwise it will be applied globally.

Comment (shashikantmaurya0007) [reply pageLayout.css:11]: but i have purposely applied to he whole page that is part of my design

Comment (SukanyaSen530) [reply pageLayout.css:11]: Okay then, Everything else look's good :)

**Pull request merged**